Sources/AppsFlyerRemoteCommand.swift

```diff
@@ -138,6 +138,11 @@ public class AppsFlyerRemoteCommand: RemoteCommand {
                         return appsFlyerInstance.logEvent(eventName, values: payload.filterVariables())
                     }
                     appsFlyerInstance.logEvent(eventName, values: eventParameters)
+                } else {
```

Comment:
@trister1997 instead of having this if else and duplicated guard let and logEvent calls, could we have a method that returns the event name (that is either $0 or $0 passed in eventCommandNames + String(standardEventName) and just call that method once before having that guard let and parameters extraction?